source/elements/oneMKL/source/domains/dft/compute_backward.rst

```diff
@@ -312,13 +318,15 @@ compute_backward (USM version)
    inout_re
       Pointer to USM allocation of sufficient capacity to store the elements
       defining the real parts of all the relevant data sequences, as
-      configured by ``desc``. Only with complex descriptors configured for in-place
+      configured by ``desc``. ``data_type`` must be single or double precision floating-point, as 
+      described by the descriptor's precision. Only with complex descriptors configured for in-place
       operations with :ref:`onemkl_dft_complex_storage_real_real`.
 
    inout_im
       Pointer to USM allocation of sufficient capacity to store the elements
       defining the imaginary parts of all the relevant data sequences, as
-      configured by ``desc``. Only with complex descriptors configured for in-place
+      configured by ``desc``. ``data_type`` must be the floating-point type 
+      described by descriptor's precision. Only with complex descriptors configured for in-place
```

Comment:
Any reason why this instance is worded differently than the other?

Reply:
This was an error on my part. I've updated it ot be consistent in [4e444ba](https://github.com/uxlfoundation/oneAPI-spec/pull/544/commits/4e444ba34dff2e1f9bd1bb1966acd4b3754b1ea6).

Reply:
I can't resolve this discussion but looks good to me, thanks!